[2025-03-15T21:15:36.960+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_data_ingestion.ingest_products_data manual__2025-03-15T21:09:19.199161+00:00 [queued]>
[2025-03-15T21:15:37.265+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_data_ingestion.ingest_products_data manual__2025-03-15T21:09:19.199161+00:00 [queued]>
[2025-03-15T21:15:37.268+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2025-03-15T21:15:37.665+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): ingest_products_data> on 2025-03-15 21:09:19.199161+00:00
[2025-03-15T21:15:38.052+0000] {standard_task_runner.py:57} INFO - Started process 506 to run task
[2025-03-15T21:15:38.240+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_data_ingestion', 'ingest_products_data', 'manual__2025-03-15T21:09:19.199161+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmp3iefx64a']
[2025-03-15T21:15:38.622+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask ingest_products_data
[2025-03-15T21:15:39.847+0000] {logging_mixin.py:149} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-15T21:15:54.580+0000] {task_command.py:410} INFO - Running <TaskInstance: postgres_data_ingestion.ingest_products_data manual__2025-03-15T21:09:19.199161+00:00 [running]> on host 8224d49c25d3
[2025-03-15T21:16:31.434+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Blessing' AIRFLOW_CTX_DAG_ID='postgres_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_products_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T21:09:19.199161+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-15T21:09:19.199161+00:00'
[2025-03-15T21:16:44.458+0000] {local_task_job_runner.py:115} ERROR - Received SIGTERM. Terminating subprocesses
[2025-03-15T21:16:51.607+0000] {process_utils.py:135} INFO - Sending Signals.SIGTERM to group 506. PIDs of all processes in the group: [506]
[2025-03-15T21:16:51.964+0000] {local_task_job_runner.py:115} ERROR - Received SIGTERM. Terminating subprocesses
[2025-03-15T21:16:52.157+0000] {process_utils.py:135} INFO - Sending Signals.SIGTERM to group 506. PIDs of all processes in the group: [506]
[2025-03-15T21:16:52.173+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 506
